ludwig/encoders/binary_encoders.py

```diff
@@ -48,9 +48,10 @@ def __init__(
         super().__init__()
         logger.debug(' {}'.format(self.name))
 
-    def call(self, inputs, training=None, mask=None):
+    def forward(self, inputs, training=None, mask=None):
         """
             :param inputs: The inputs fed into the encoder.
                    Shape: [batch x 1], type tf.float32
         """
-        return {'encoder_output': tf.cast(inputs, dtype=tf.float32)}
+        #inputs = inputs.type(torch.float32)
+        return {'encoder_output': inputs}
```

Comment:
Could you also add `input_shape` and `output_shape` properties for the class? Example here for [category encoders](https://github.com/ludwig-ai/ludwig/blob/ee17666addc269cb022272c55edba58b6dbaff97/ludwig/encoders/category_encoders.py#L85).

Reply:
added